posthog/models/filters/test/__snapshots__/test_filter.ambr

```diff
@@ -472,7 +472,7 @@
          "posthog_filesystem"."created_at",
          "posthog_filesystem"."created_by_id"
   FROM "posthog_filesystem"
-  WHERE ("posthog_filesystem"."ref" = '75'
+  WHERE ("posthog_filesystem"."ref" = '85'
```

Comment:
@mariusandra I think we might need to make these values consistent somehow otherwise the snapshots will change regularly

Reply:
oh wow... will take a look

Reply:
I hope this will do the trick: https://github.com/PostHog/posthog/pull/30436